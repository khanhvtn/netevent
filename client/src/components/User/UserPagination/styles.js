import { makeStyles } from '@material-ui/core/styles';

export default makeStyles((theme) => ({
    paginationWrapper: {
        display: 'flex',
        flex: '1 1 100%',
        justifyContent: 'flex-end',
        alignItems: 'center',
        paddingTop: 8,
        paddingBottom: 8,
        minHeight: 52,
        paddingRight: 2,
        backgroundColor: theme.palette.background.paper
    },
    selectRowNumWrapper: {
        display: 'flex',
        justifyContent: 'center',
        alignItems: 'center',
        marginRight: 16,
    },
    selectRowNum: {
        marginLeft: 8,
        backgroundColor: 'transparent',
        outline: 'none',
    },
    pagination: { marginRight: 8 },
}));
