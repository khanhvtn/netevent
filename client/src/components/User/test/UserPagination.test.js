import React from 'react'
import { Provider } from 'react-redux'
import { renderer } from 'react-test-renderer'
import configureStore from 'redux-mock-store'
import beforeEach from 'jest-wrap'
import expect from 'expect'
import toJson from 'enzyme-to-json'

import UserPagination from '../UserPagination/UserPagination'

const mockStore = configureStore([])
describe('UserPagination', () => {
    let store;
    let component;

    beforeEach(() => {
        store = mockStore({
            myState: 'User Pagination'
        })

        store.dispatch = jest.fn()

        component = renderer.create(
            <Provider store={store}>
                <UserPagination />
            </Provider>
        )
    })
    
    it('should render with given state from Redux store', () => {
        expect(toJson(component)).toMatchSnapshot()
    })
})