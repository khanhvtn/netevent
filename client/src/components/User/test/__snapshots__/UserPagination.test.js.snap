// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserPagination should render with given state from Redux store 1`] = `undefined`;
